src/baseline_automatic_pipeline_system/coreference.py

```diff
@@ -182,17 +185,32 @@ def eval_entity_coref_with_gold_graph(gold_graph):
                                       entity.mentions.values()]
     entity_clusters = cluster_entity_mentions(entity_mentions_for_clustering)
     entity_clusters_for_eval = [set([item[0] for item in cluster]) for cluster in entity_clusters]
-    curr_entity_scores = eval_clusters(entity_clusters_for_eval, gold_graph)
+
+    graph1_ent_mentions, graph2_ent_mentions = extract_mentions(entity_clusters_for_eval, gold_graph)
+    curr_entity_scores = eval_clusters(graph1_ent_mentions, graph2_ent_mentions)
+
+    # create base line GS file
+    create_gs_test_result(graph2_ent_mentions)
+
     return curr_entity_scores
 
 
-if __name__ == '__main__':
+def create_gs_test_result(gold_graph):
```

Comment:
Please add documentation for this function.

Reply:
Done, will push with all changes